frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -514,51 +515,91 @@ const UpdatesType: React.FC<UpdatesTypeProps> = ({ children }) => {
   const { t } = useTranslation();
-  return (
-    <UpdatesGroup divided>
+  return hideIfComplete && percentMCPNodes === 100 ? null : (
+    <UpdatesGroup divided={divided}>
       <UpdatesType>
-        <Link to="/k8s/cluster/nodes?rowFilter-node-role=worker">
-          {t('cluster-settings~Worker nodes')}
+        <Link to={`/k8s/cluster/nodes?rowFilter-node-role=${machineConfigPool.metadata.name}`}>
+          {t('cluster-settings~{{name}} {{resource}}', { name, resource: NodeModel.labelPlural })}
         </Link>
-        <Popover
-          bodyContent={
-            <>
-              {t(
-                'cluster-settings~Worker nodes may continue to update after the update of master nodes and operators are complete.',
-              )}
-            </>
-          }
-        >
-          <Button
-            variant="plain"
-            aria-label="Help"
-            className="co-help-popover-button co-help-popover-button--space-l"
+        {name !== 'Master' && (
+          <Popover
+            bodyContent={
+              <>
+                {t(
+                  'cluster-settings~{{name}} {{resource}} may continue to update after the update of Master {{resource}} and {{resource2}} are complete.',
+                  {
+                    name,
+                    resource: NodeModel.labelPlural,
+                    resource2: ClusterOperatorModel.labelPlural,
+                  },
+                )}
+              </>
+            }
           >
-            <OutlinedQuestionCircleIcon />
-          </Button>
-        </Popover>
+            <Button
+              variant="plain"
+              aria-label="Help"
+              className="co-help-popover-button co-help-popover-button--space-l"
+            >
+              <OutlinedQuestionCircleIcon />
+            </Button>
+          </Popover>
+        )}
       </UpdatesType>
       <UpdatesBar>
         <Progress
-          title={t('cluster-settings~{{updatedWorkerNodes}} of {{totalWorkerNodes}}', {
-            updatedWorkerNodes,
-            totalWorkerNodes,
+          title={t('cluster-settings~{{updatedMCPNodes}} of {{totalMCPNodes}}', {
+            updatedMCPNodes,
+            totalMCPNodes,
           })}
-          value={!_.isNaN(percentWorkerNodes) ? percentWorkerNodes : null}
+          value={!_.isNaN(percentMCPNodes) ? percentMCPNodes : null}
           size={ProgressSize.sm}
-          variant={percentWorkerNodes === 100 ? ProgressVariant.success : null}
+          variant={percentMCPNodes === 100 ? ProgressVariant.success : null}
         />
       </UpdatesBar>
     </UpdatesGroup>
   );
 };
 
+const OtherNodes: React.FC<OtherNodesProps> = ({
+  hideIfComplete,
+  machineConfigPools,
+  updateStartedTime,
+}) => {
+  const otherNodes = machineConfigPools
+    .filter((mcp) => mcp.metadata.name !== 'master' && mcp.metadata.name !== 'worker')
+    .sort((a, b) => (a.metadata.creationTimestamp > b.metadata.creationTimestamp ? 1 : -1));
```

Comment:
This should return `0` if they are the same. I'd probably just use

```suggestion
    .sort((a, b) => a.metadata.creationTimestamp.localeCompare(b.metadata.creationTimestamp));
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare